lmcache/config.py

```diff
@@ -7,16 +7,20 @@
 
 @dataclass
 class LMCacheEngineMetadata:
-    """name of the LLM model"""
-
+    """ name of the LLM model """
     model_name: str
+    
     """ world size when running under a distributed setting """
     world_size: int
+    
     """ worker id when running under a distributed setting """
     worker_id: int
+    
     """ the format of kv tensors """
     fmt: str
 
+    """ the data type of kv tensors """
+    dtype: str
```

Comment:
Is there any reason to support `dtype = "auto"` in LMCache? 
I think in any case, the LMCache engine needs to know an explicit dtype instead of auto?
@KuntaiDu @YaoJiayi Please chime in if you have any opinion about this

Reply:
@ApostaC 

Yes, what we agreed on last time is to make sure there is no "auto" option for dtype in lmcache metadata, i.e. dtype must be an explicit type. Currently, this is achieved by overwriting any "auto" value of dtype in lmcache metadata with the dtype of the model in lmcache_vllm/vllm_injection.py.